### ISSUE TITLE
Explore the process of migrating to using PIH config as the CHT demo config

### ISSUE THREAD

**Issue by antonykhaemba** [medic/cht-core#9487]
Here is the proposed plan:

- [ ] Set up CHT 4.x docker, upload the PIH config and test to make sure nothing is broken.
- [ ] Update the PIH config and unit tests to align with the latest CHT version.
- [ ] Upgrade the PIH config to the target latest release.
- [ ] Test the config updates locally.
- [ ] Upload the updated PIH configs to this[ instance](https://test-demo-cht.dev.medicmobile.org/)
- [ ] Review and complete the translations to align with the current demo app translations.
- [ ] Conduct and end to test workflows testing using the test-demo instance.
- [ ] Upload the updated PIH config to the [demo instance](https://demo-cht.dev.medicmobile.org/)
- [ ] Run an end to end testing of the workflows.
- [ ] Test the demo account creation process. Try to create a new user account and validate if the new user can access the updated CHT demo workflows, also verify if an already existing user can access the demo app.

**Comment by garethbowen** [comment]
@antonykhaemba Sounds good. We should also decide how we're going to maintain the new demo config going forward. Essentially it sounds like you're proposing forking the config because some of those changes won't be backwards compatible. I think this is the right approach so we can continue to evolve the demo config separately from the PIH config. Assuming this is the case, will it be committed directly to the cht-core repo, or a separate repo?

An alternative would be to publish it as a reference app as well, but that's probably out of scope for now.

**Comment by antonykhaemba** [comment]
Thanks @garethbowen for your feedback.

The PIH app is already [published](https://docs.communityhealthtoolkit.org/building/examples/pih/) as a reference app.

**Comment by garethbowen** [comment]
@antonykhaemba Yes but we will be making changes to it over time (eg: supporting 4.x) which will break backwards compatibility. Is that ok, or should we fork it as a separate demo app?

**Comment by antonykhaemba** [comment]
We can folk it as a separate demo app @garethbowen

**Comment by mrjones-plip** [comment]
As we're now [removing the demo config](https://github.com/medic/cht-core/issues/11353), and this ticket is now years old, closing it.

@antonykhaemba - please reopen if we still want to follow up on this!

**Comment by jkuester** [comment]
> As we're not https://github.com/medic/cht-core/issues/11353, and this ticket is now years old, closing it.

Just for the record, I think you mean "now" instead of "not".  😅

**Comment by mrjones-plip** [comment]
yes - thanks!  I've fixed it, it was "As we're **_not_** removing the demo config" and it [now reads](https://github.com/medic/cht-core/issues/9487#issuecomment-5304098805) "As we're **_now_** removing the demo config"
